### PR TITLE
Add Mega-Linter in "Code Review tools" sections

### PR DESCRIPTION
**Pull request by nvuillam**
### What I did 

I added a link to [Mega-Linter](https://nvuillam.github.io/mega-linter) in the list of Code Review Tools

### How I did it

By updating MD file :)

### How to verify it

Read the readme file :)

### CODE REVIEW

File: docs/INTEGRATION.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ Then add the CI step on codeship-steps.yml with the path of the dockerfile
     - service: hadolint
       command: hadolint /test/Dockerfile
 ```
+### Mega-Linter
+
+[Mega-Linter](https://nvuillam.github.io/mega-linter/) aggregates **70** linters, including [hadolint](
```
Comment (zemanlx) [inline docs/INTEGRATION.md:34]: IMHO this text is short enough that using bold text is unnecessary.

File: docs/INTEGRATION.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ Then add the CI step on codeship-steps.yml with the path of the dockerfile
...
+https://nvuillam.github.io/mega-linter/descriptors/dockerfile_hadolint/) out of the box
```
Comment (zemanlx) [inline docs/INTEGRATION.md:35]: Missing full stop at the end of the text.

**Review by zemanlx — CHANGES_REQUESTED** [review]

File: docs/INTEGRATION.md (hunk 1 above)
Comment (nvuillam) [reply docs/INTEGRATION.md]: updated

File: docs/INTEGRATION.md (hunk 2 above)
Comment (nvuillam) [reply docs/INTEGRATION.md]: updated

**Comment by nvuillam** [comment]
@zemanlx I performed the requested updates :) 

I've been lazy and did it directly on github, so there are new commits
If you want only one commit, you can "Squash & merge", or I can make a new PR, as you prefer :)

**Review by zemanlx — APPROVED** [review]

**Comment by lorenzo** [comment]
Thanks!

**Comment by nvuillam** [comment]
@lorenzo > Thanks to you for accepting the PR :)
I see you have separate Actions for markdown and haskell lints, would you like me to setup Mega-Linter so you'll have all in one action, + other languages & used formats that are not linted yet in your CI ?

**Pull request merged**